RHEL6_7/packages/RemovedPackages/check

```diff
@@ -68,8 +85,9 @@ echo \
 If a package not signed by Red Hat requires these packages, you may need to ask your
 vendor to provide an alternative solution, or you may get the missing package from
 other sources than Red Hat Enterprise Linux.
+
 " >>solution.txt
-[ $found -eq 1 ] && log_high_risk "After upgrading to Red Hat Enterprise Linux 7, there are still some el6 packages left. Add the '--cleanup-post' option to redhat-upgrade-tool to remove them automatically."
+[ $found -eq 1 ] && log_high_risk "After upgrading to Red Hat Enterprise Linux 7, there are still some el6 packages left. Add the '--cleanup-post' option to redhat-upgrade-tool to remove them automatically. You can also run manually the $NOAUTO_POST_DIR/$NOAUTO_SCRIPT_NAME script after the upgrade to remove only old native packages with broken dependencies."
```

Comment:
- *Red Hat Enterprise Linux 7, there are still some el6* **->** *Red Hat Enterprise Linux (RHEL) 7, there are still some RHEL 6*
- *run manually the XY script* **->** *run the XY script manually*
- *only old native packages with broken dependencies* **->** *only those RHEL 6 packages that have broken dependencies*

Reply:
fixed